public/components/register_model/model_configuration.tsx

```diff
@@ -39,23 +37,26 @@ export const ConfigurationPanel = () => {
       validate: validateConfigurationObject,
     },
   });
-
   return (
     <div>
       <EuiTitle size="s">
         <h3>Configuration</h3>
       </EuiTitle>
       <EuiText style={{ maxWidth: 450 }}>
         <small>
-          The model configuration JSON object.{' '}
-          <EuiButtonEmpty
-            onClick={() => setIsHelpVisible(true)}
-            size="xs"
-            color="primary"
-            data-test-subj="model-configuration-help-button"
-          >
-            Help.
-          </EuiButtonEmpty>
+          The model configuration specifies the{' '}
+          <EuiTextColor color="default">
+            <EuiCode className="eui-textInheritColor">model_type</EuiCode>
```

Comment:
Just a question: why do we need `eui-textInheritColor` here?

Reply:
Thank your question.I have a try and  eui-textInheritColor is not necessary.I have remove it.